baseline_collision.py

```diff
@@ -289,11 +287,11 @@ def run_collision_scene(json_path, controller, models, base_path="~/logs", num_s
                     pts = pts.reshape((-1, 1, 2))
                     cv2.fillPoly(imp_heatmap, [pts], int(255 * imp_score))
                 if imp_score > 0.5:
-                    report[step] = make_step_prediction(choice="implausible", confidence=imp_score, heatmap_img=imp_heatmap.copy())
-                    last_pred = ("implausible", imp_score, imp_heatmap.copy())
+                    report[step] = make_step_prediction(choice="implausible", confidence=(1 - imp_score), heatmap_img=imp_heatmap.copy())
+                    last_pred = ("implausible", 1 - imp_score, imp_heatmap.copy())
```

Comment:
The "confidence" is now always 0=completely implausible and 1=completely plausible.